### PR TITLE
Dos tests

### PR DESCRIPTION
**Pull request by pasmant**
Make dos tests more robust

### CODE REVIEW

File: tests/suite/test_virtual_server_dos.py [hunk 1]
```diff
@@ -206,6 +215,12 @@ def test_dos_vs_logs(
         print(f"log_loc: {log_loc} syslog_pod: {syslog_pod} namespace: {ingress_controller_prerequisites.namespace}")
         log_contents = get_file_contents(kube_apis.v1, log_loc, syslog_pod, ingress_controller_prerequisites.namespace)
 
+        # print("\n===================== ADMD Logs Start =====================")
```
Comment (vepatel) [inline tests/suite/test_virtual_server_dos.py:218]: @pasmant can you please remove the commented code if not needed?

Comment (pasmant) [reply tests/suite/test_virtual_server_dos.py:218]: There is any log level in pytest?

Comment (pasmant) [reply tests/suite/test_virtual_server_dos.py]: Remove the logs

File: tests/suite/test_virtual_server_dos.py [hunk 2]
```diff
@@ -133,6 +141,7 @@ def fin():
         delete_dos_policy(kube_apis.custom_objects, pol_name, test_namespace)
         delete_dos_logconf(kube_apis.custom_objects, log_name, test_namespace)
         delete_dos_protected(kube_apis.custom_objects, protected_name, test_namespace)
+        clean_good_bad_clients()
         # delete_items_from_yaml(kube_apis, src_webapp_yaml, test_namespace)
```
Comment (vepatel) [inline tests/suite/test_virtual_server_dos.py:145]: @pasmant  more commented code here, can you please have a look?

Comment (pasmant) [reply tests/suite/test_virtual_server_dos.py]: Done

**Review by vepatel — APPROVED** [review]

File: tests/suite/test_dos.py [hunk 3]
```diff
@@ -233,74 +243,6 @@ def test_dos_sec_logs_on(
         assert f'vs_name="{test_namespace}/dos-protected/name"' in log_contents
         assert "bad_actor" in log_contents
 
-    def test_dos_under_attack_no_learning(
```
Comment (lucacome) [inline tests/suite/test_dos.py:236]: we don't need this anymore?

**Pull request merged**